### ISSUE TITLE
Invalid configuration setup causes config load failure

### ISSUE THREAD

**Issue by Linguardium** [Xalcon/TorchMaster#240]
https://github.com/Xalcon/TorchMaster/blob/be86cf9fb6974d4c1abc1a4a26381d2477b39e07/neoforge/src/main/java/net/xalcon/torchmaster/TorchmasterNeoforgeConfig.java#L88-L96

i believe these should be using the `defineListAllowEmpty` rather than the `define`

Alternatively, your validator should validate the actual config value ( a list) rather than the element values

**Comment by Xalcon** [comment]
Sorry for the delay, I had some issues understanding what you actually want to get at.
I guess you mean the game crashes when you assign a string to the list rather than a list of strings? Yeah, that was something I didn't test. The config code is quite old, and I'm not sure when they introduced the list defines.

I've switched to `defineListAllowEmpty` and I'm providing a validator for each entry. I've also updated the comments to give a bit more details on how the syntax works in toml. I'll release these changes when I port some features from 1.20 to 1.21


Thanks for reporting the issue
